packages/phpstan-rules/tests/Rules/ForbiddenMethodCallByTypeInLocationRule/config/configured_rule.neon

```diff
@@ -0,0 +1,8 @@
+services:
+    -
+        class: Symplify\PHPStanRules\Rules\ForbiddenMethodCallByTypeInLocationRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            forbiddenTypeInLocations:
+                View\Helper: ['Controller']
+                Symplify\PHPStanRules\Tests\Rules\ForbiddenMethodCallByTypeInLocationRule\Fixture\View\Helper\NumberHelper: ['Controller']
```

Comment:
We usually use location first, then rules.

Could you switch that?

```yaml
Controller:
    - View\Helper
    - Symplify\PHPStanRules\Tests\Rules\ForbiddenMethodCallByTypeInLocationRule\Fixture\View\Helper\NumberHelper
```

Reply:
I will try